src/r4300/new_dynarec/new_dynarec.c

```diff
@@ -4970,7 +4970,7 @@ static void do_ccstub(int n)
     if(itype[i]==RJUMP)
     {
       int r=get_reg(branch_regs[i].regmap,rs1[i]);
-      if(rs1[i]==rt1[i+1]||rs1[i]==rt2[i+1]) {
+      if((rs1[i]==rt1[i+1]||rs1[i]==rt2[i+1])&&(rs1[i]!=0)) {
```

Comment:
Lol this won't make a big difference. Also the case where rs1==0 is unlikely to happen very often.